cli/cmdlineparser.cpp

```diff
@@ -1191,14 +1191,17 @@ CmdLineParser::Result CmdLineParser::parseFromArgs(int argc, const char* const a
                 mSettings.libraries.emplace_back("windows");
             }
             if (projectType == ImportProject::Type::MISSING) {
+                project.printErrors();
```

Comment:
it seems better to output the error messages using `mLogger::printError` instead of writing them to `std::cout`. Overall it should be avoided to use std::cout directly; if the user open the gui from a terminal then imho it will be weird to output error/warning messages in the terminal..